modin/engines/base/io/text/text_file_dispatcher.py

```diff
@@ -258,3 +259,157 @@ def _read_rows(
             rows_read += 1
 
         return outside_quotes, rows_read
+
+    # _read helper functions
+    @classmethod
+    def rows_skipper_builder(cls, f, quotechar, is_quoting):
+        def skipper(n):
+            if n == 0 or n is None:
+                return 0
+            else:
+                return cls._read_rows(
+                    f,
+                    quotechar=quotechar,
+                    is_quoting=is_quoting,
+                    nrows=n,
+                )[1]
+
+        return skipper
+
+    @classmethod
+    def _define_header_size(
+        cls,
+        header: Union[int, Sequence[int], str, None] = "infer",
+        names: Union[Sequence, None] = None,
```

Comment:
Can we use `Optional` here as follows?
```suggestion
        names: Optional[Sequence] = None,
```